### ISSUE TITLE
Support legacy v0 (Blink) Shadow DOM

### ISSUE THREAD

**Issue by treshugart** [skatejs/skatejs#561]
This would improve performance and support existing Chrome and also the current Polymer Shadow DOM polyfill.

The only internals that would really need updating are the `vdom` module so that when you render a slot, it abstracts away the fact that it's using a `<content>` element instead of a `<slot>` element like so:

```html
<content select="[slot="slot-name"]">default content</content>
```

The other part would be to support it when creating a shadow root in the `src/lifecycle/render` module:

```js
if (elem.attachShadow) {
  ...
} else if (elem.createShadowRoot) {
  ...
}
```

**Comment by treshugart** [comment]
Coincides with #562.

**Comment by treshugart** [comment]
This has been implemented at the same time as #562.
